src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -466,4 +466,9 @@ public net.minecraft.world.level.levelgen.feature.trunkplacers.TrunkPlacerType <
 protected net.minecraft.world.level.portal.PortalForcer f_77648_ # level
 public net.minecraft.world.level.storage.LevelResource <init>(Ljava/lang/String;)V # constructor
 private-f net.minecraft.world.level.storage.loot.LootPool f_79028_ # rolls
-private-f net.minecraft.world.level.storage.loot.LootPool f_79029_ # bonusRolls
\ No newline at end of file
+private-f net.minecraft.world.level.storage.loot.LootPool f_79029_ # bonusRolls
+public net.minecraft.client.renderer.texture.atlas.sources.PalettedPermutations <init> # constructor
+public net.minecraft.client.renderer.texture.atlas.sources.PalettedPermutations$PalettedSpriteSupplier
```

Comment:
Missing the AT for the constructor. Regardless of `PalettedPermutations.PalettedSpriteSupplier` being a record, the class access level and constructor access level are still separate things (even though you cannot see the latter in the source)

Reply:
I completely missed this yesterday and also had to check to be sure: constructor ATs are essentially standard method ATs and require the method descriptor to be specified. The four constructor ATs are currently invalid and don't do anything